common.yaml

```diff
@@ -0,0 +1,286 @@
+openapi: 3.0.3
+info:
+  title: Common
+  description: Cloudflare Common API Components
+  version: 4.0.0
+paths: {}
+components:
+  schemas:
+    uuid:
+      description: UUID
+      readOnly: true
+      type: string
+      maxLength: 36
+      example: f174e90a-fafe-4643-bbbc-4a0ed4fc8415
+    identifier:
+      description: Identifier
+      type: string
+      maxLength: 32
+      readOnly: true
+      example: 023e105f4ecef8ad9ca31a8372d0c353
```

Comment:
for future: let's add some [semgrep](https://semgrep.dev/) rules around this so that we don't have variances in the documentation of identifiers (same for UUIDs and domains)

Reply:
https://github.com/cloudflare/cloudflare-go/blob/master/.semgrep.yml are some examples of where we do this in other libraries as a starting point.